src/screens/Game/GamePage/index.tsx

```diff
@@ -193,6 +195,9 @@ export default function GamePage(): JSX.Element | null {
                   <div className="title">{title}</div>
                   <div className="infoWrapper">
                     <div className="developer">{developer}</div>
+                    {namespace == 'ue' && (
```

Comment:
Maybe use a variable for this check to make It more readable since you're using in other places as well :)

Reply:
If you don't want for the asset to have settings, maybe its better to disable the settings button. Make more sense I guess.